{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -6,7 +6,7 @@ wheel==0.30.0
 
 
 # Conservative Django
-django==1.11.8 # pyup: <2.0
+django==2.0
```

Comment:
the next django version is 2.1 and not 3.0
`django==2.0 # pyup: < 2.1`

Reply:
`2.x` is unlikely to violate semver introducing breaking changes -- that's why I metioned `3.0`. But, yes, no harm restricting it further.

Reply:
I think it's prudent to limit it to 2.1 because is expected to have backward incompatibilities

https://docs.djangoproject.com/en/dev/internals/release-process/#release-cadence

https://docs.djangoproject.com/en/dev/releases/2.1/#backwards-incompatible-2-1

Reply:
Didn't know that, thanks.